### PR TITLE
feat: add tests for "Roles" tab

### PR DESCRIPTION
**Pull request by Vorobeyko**
## Description

✨ What’s New
- 👥 Roles tab tests added. Ensures coverage for all role-related functionality and edge cases.
- 🔑 Keys tab tests relocated. Moved existing Keys tests into a dedicated tests/keys/ folder for cleaner structure.
- ⚙️ WTM updated. Bumped WTM to the latest version, resolving the Mainnet connection problem. 🌐✅

## Checklist

- [ ] Requires other services change
- [ ] Affects to other services
- [ ] Requires dependency update
- [ ] Automated tests
- [ ] Looks good on large screens
- [ ] Looks good on mobile

### CODE REVIEW

**Review by exromany — DISMISSED** [review]

**Review by exromany — DISMISSED** [review]

File: tests/widget/operatorWithValidator/roles/rewardsAddress/transactions.spec.ts [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+            await rewardsAddressPage.page.waitForSelector(
+              `text=Proposed request for rewards address has been revoked`,
+              { timeout: STAGE_WAIT_TIMEOUT },
+            );
+
+            const { txModal } = widgetService.rolesPage;
+            await expect(txModal.description).toContainText('Address stays');
+
+            await expect(txModal.description).toContainText(
+              trimAddress(
```
[the remark this answers is not in the document]
Comment (Vorobeyko) [reply tests/widget/operatorWithValidator/roles/rewardsAddress/transactions.spec.ts]: no) because i need to trim it

File: tests/widget/operatorWithValidator/roles/managerAddress/proposedAddress.spec.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    });
+
+    test.afterAll(async ({ widgetService }) => {
+      await widgetService.rolesPage.managerAddressPage.open();
+      await widgetService.page.waitForTimeout(1000);
+      await widgetService.rolesPage.managerAddressPage.revokePendingRole();
+    });
+
+    test(
+      qase(219, 'Verify UI elements in the "Manager Address" tab'),
```
[the remark this answers is not in the document]
Comment (Vorobeyko) [reply tests/widget/operatorWithValidator/roles/managerAddress/proposedAddress.spec.ts:32]: Basically, we’re just checking the UI with common actions here. I didn’t separate it to small tests.

**Review by itaven — APPROVED** [review]
